### ISSUE TITLE
function featuresFromGeoJsonMainThread is not working

### ISSUE THREAD

**Issue by raycastsant** [synw/geojson#36]
Hello

I'm developing a proyect, and I need to represent on the map some lines that comes in GeoJson format.
In Android platform I can do this well, but in the Web this is not working.
I've implemented the featuresFromGeoJsonMainThread () funtion  but returns empty values.
Please can you help me?

**Comment by FaFre** [comment]
I have the same problem, is there any solution already for that?

**Comment by raycastsant** [comment]
Hello

I downloaded the source code from https://github.com/synw/geojson, and commented line #178 in here: https://github.com/synw/geojson/blob/master/lib/src/geojson.dart.

I hope this help.

**Comment by FaFre** [comment]
Thanks, with that it's working - but only for small data. With bigger data the stream will not process to end it seems....

**Comment by FaFre** [comment]
Yeah, there was a await missing, which didn't affect smaller data I guess. However, in my short tests, the single threaded version even feels a bit faster on native Linux application.

**Comment by sannchesda** [comment]
> Hello
> 
> I downloaded the source code from https://github.com/synw/geojson, and commented line #178 in here: https://github.com/synw/geojson/blob/master/lib/src/geojson.dart.
> 
> I hope this help.

Wow it works for me
